app/operators/fetal_mri_3d_brain_recon_operator.py

```diff
@@ -33,28 +33,26 @@ def compute(self, op_input: InputContext, op_output: OutputContext, context: Exe
 
         # Run motion corrected reconstruction
         if not is_local_testing:
-            if torch.cuda.is_available():
-                cnn_mode = "1"
-                logging.info("SVRTK reconstruction using GPU mode ...")
-            if not torch.cuda.is_available():
-                cnn_mode = "-1"
-                logging.info("SVRTK reconstruction using CPU mode ...")
-
-            motion_correction_mode = "-1"  # -1 minor, 1 severe
-            logging.info("SVRTK reconstruction using Minor motion correction mode ...")
+#            if torch.cuda.is_available():
+#                cnn_mode = "1"
+#                logging.info("SVRTK reconstruction using GPU mode ...")
+#            if not torch.cuda.is_available():
+#                cnn_mode = "-1"
+#                logging.info("SVRTK reconstruction using CPU mode ...")
+#
+#            motion_correction_mode = "-1"  # -1 minor, 1 severe
+#            logging.info("SVRTK reconstruction ...")
```

Comment:
I'm not keen on commenting this out. I see you've shifted the cnn_mode/motion_correction_mode variables into the bash script. The problem is that it puts a hard requirement on GPU. In testing, I've found it useful to run in CPU mode.

Leave it for now, but something we can discuss.

Reply:
the current docker is fully CPU based by default